.yamllint

```diff
@@ -20,3 +20,4 @@ ignore: |
   .github/
   .gitlab/
   .gitlab-ci.yml
+  *etc/kayobe/kolla/config/prometheus/prometheus.yml.d/70-oscapacity.yml
```

Comment:
This file was renamed

Reply:
This was due to `70.oscapacity.yml.j2` not being templated as `70-oscapacity.yml` during deployment which caused Prometheus to not pick up the scrape job. I removed the .j2 extension and added it to the yamllint to stop ci failing it for the raw tags.

Reply:
```suggestion
  *etc/kayobe/kolla/config/prometheus/prometheus.yml.d/70-os-capacity.yml
```

Reply:
hmm, I thought it had been renamed to 70-os-capacity.yml, but looks like I was wrong.